api/src/main/java/org/eclipse/microprofile/faulttolerance/Fallback.java

```diff
@@ -36,6 +36,17 @@
  * <li>If fallbackMethod is specified, invoke the method specified by the fallbackMethod on the same class.</li>
  * <li>If both are specified, the {@link org.eclipse.microprofile.faulttolerance.exceptions.FaultToleranceDefinitionException} must be thrown.</li>
  * </ol>
+ * <p>
+ * When a method returns and the Fallback policy is present, the following rules are applied:
+ * <ol>
+ * <li>If the method returns normally (doesn't throw), the result is simply returned.
```

Comment:
In this part "(doesn't throw)" we can add (doesn't throw an exception)

Reply:
fixed